doc/gpumd/input_parameters/mc.rst

```diff
@@ -0,0 +1,102 @@
+.. _kw_mc:
+.. index::
+   single: mc (keyword in run.in)
+
+:attr:`mc`
+==========
+
+The :attr:`mc` keyword is used to invoke Monte Carlo (:term:`MC`) simulation, usually in combination with :term:`MD` simulation. 
+Three :term:`MC` ensembles are supported, including the canonical, the semi-grand canonical (:term:`SGC`), and the variance-constrained semi-grand canonical (:term:`VCSGC`) [Sadigh2012a]_ [Sadigh2012b]_ ones. 
+
+This keyword can only be used when the potential is :term:`NEP`.
+
+Syntax
+------
+
+:attr:`canonical`
+^^^^^^^^^^^^^^^^^
+If the first parameter is :attr:`canonical`, it means to use the canonical :term:`MC` ensemble.
+
+It can be used in the following way::
+
+    mc canonical <md_steps> <mc_trials> <T_i> <T_f> [group <grouping_method> <group_id>]
+
+:attr:`sgc`
+^^^^^^^^^^^
+If the first parameter is :attr:`sgc`, it means to use the :term:`SGC` :term:`MC` ensemble.
+
+It can be used in the following way::
+
+    mc sgc <md_steps> <mc_trials> <T_i> <T_f> <num_species> {species_0 mu_0 species_1 mu_1 ...} [group <grouping_method>  <group_id>]
+
+:attr:`vcsgc`
+^^^^^^^^^^^
+If the first parameter is :attr:`vcsgc`, it means to use the :term:`VCSGC` :term:`MC` ensemble.
+
+It can be used in the following way::
+
+    mc vcsgc <md_steps> <mc_trials> <T_i> <T_f> <num_species> {species_0 phi_0 species_1 phi_1 ...} kappa [group <grouping_method>  <group_id>]
+
+* :attr:`mc_trials` :term:`MC` trials are performed every :attr:`md_steps` :term:`MD` steps.
+
+* The instant temperature for the :term:`MC` ensemble will linearly change from attr:`T_i` to attr:`T_f`.
+
+* :attr:`num_species` is the number of species to be involved in the :term:`SGC` or :term:`VCSGC` ensemble. It is required to be no less than 2 and no larger than 4.
+
+* For the :term:`SGC` ensemble, after specifying the number of species to be involved, the chemical symbols and chemical potentials (in units of eV) for these species should be listed, in an arbitrary order.
+
+* For the :term:`VCSGC` ensemble, after specifying the number of species to be involved, the chemical symbols and (dimensionless) :math:`\phi` parameters for these species should be listed, in an arbitrary order. One then needs to specify the (dimensionless) :math:`\kappa` parameter. The :math:`\phi` and :math:`\kappa` parameters constrain the average and variance of the species concentrations, respectively. (Do we need to cite papers for the exact definitions of these parameters?)
```

Comment:
```suggestion
* For the :term:`VCSGC` ensemble, after specifying the number of species to be involved, the chemical symbols and (dimensionless) :math:`\phi` parameters for these species should be listed.
  The species can be listed in arbitrary order.
* Next one needs to specify the (dimensionless) :math:`\kappa` parameter.
* The :math:`\phi` and :math:`\kappa` parameters constrain the average and variance of the species concentrations, respectively.
*  One can usually achieve a sampling of the full composition range by varying :math:`\phi` between -1.2 and +1.2.
* Typically a :math:`\kappa` value of 100 is suitable.
  If the concentration fluctuations are too large (e.g., deep with miscibility gaps) one should increase this value.
```

The choice of parameters that we use here differs from the original papers in terms of normalization.
The "modern" choice is largely system agnostic.
The expressions can be found, e.g., in [this paper](http://doi.org/10.1016/j.actamat.2021.116893).